webpack/assets/javascripts/react_app/components/HostDetails/Templates/CardItem/CardTemplate/index.js

```diff
@@ -17,8 +21,21 @@ import {
   GridItem,
 } from '@patternfly/react-core';
 
-const CardItem = ({ content, header, children }) => {
+const CardItem = ({
+  content,
+  header,
+  children,
+  dropdownItems,
+  dropdownSelectHandler,
+}) => {
   const [activeAccordion, setActiveAccordion] = useState(0);
+  const [dropdownVisibility, setDropdownVisibility] = useState(false);
+
+  const onDropdownSelect = event => {
+    setDropdownVisibility(prevState => !prevState);
+    dropdownSelectHandler(event);
+  };
+  const onDropdownToggle = isOpen => setDropdownVisibility(isOpen);
```

Comment:
Should this be
```suggestion
  const onDropdownToggle = isOpen => setDropdownVisibility(!isOpen);
```
or even
```suggestion
  const onDropdownToggle = _isOpen => setDropdownVisibility(!dropdownVisibility);
```
?

Reply:
No need, the onToggle function from KebabToggle already gives the opposite state
a bit odd, but this is the implementation in PF

https://www.patternfly.org/v4/components/dropdown